engine-tests/src/main/java/org/terasology/testUtil/ModuleManagerFactory.java

```diff
@@ -36,15 +37,22 @@ private ModuleManagerFactory() {
 
     public static ModuleManager create() throws Exception {
         ModuleManager moduleManager = new ModuleManagerImpl("");
-        return LoadModuleEnvironment(moduleManager);
+        return loadModuleEnvironment(moduleManager);
     }
 
+    /**
+     * Creates a ModuleManager with ModuleInstallManager configured to reference masterServerAddress
```

Comment:
What is the `masterServerAddress` supposed to provide? Can your describe in a few words what the server has do provide? REST interface? Providing files in some specific structure? A maven repository?

Reply:
added more detail